andela/travel_tool_front
# #160196032 Approve and Reject request(By Requester's Manager)

#### What does this PR do?
This PR adds the functionality to allow a Requesters Manager to approve requests.

#### Description of Task to be completed?
- write tests for request status update
- create reducer to make request status update
- create saga to consume request status update api endpoint on the backend
- add functionality for request details to show on approvals table when a request is clicked
- add functionality to status buttons to update status when clicked by requesters manager

#### How should this be manually tested?
- Clone the repo - https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - cd travel_tool_front
- Install dependencies and start the app - make start
- Create a new request on My Requests tab and ensure you're listed as your manager
- Log in, click Requests tab on the left sidebar and select My Approvals
- Click an open request
- Any button out of `Approve` and `Reject` can be clicked to update the status of the request

#### Any background context you want to provide?
This PR works closely with https://github.com/andela/travel_tool_back/pull/38 to resolve the error that shows `You have no approvals at the moment` even when there are data in `Approvals table` on the database

#### What are the relevant pivotal tracker stories?
[#160196032](https://www.pivotaltracker.com/story/show/160196032)

#### Screenshots (if appropriate)

<img width="1438" alt="screen shot 2018-09-06 at 11 47 28 am" src="https://user-images.githubusercontent.com/23608060/45153157-f5ca8300-b1ca-11e8-81f9-9c85dbabf4c7.png">
<img width="1440" alt="screen shot 2018-09-06 at 11 46 31 am" src="https://user-images.githubusercontent.com/23608060/45153112-cc115c00-b1ca-11e8-9396-60da79684c04.png">


#### Questions:
